docs/tutorials/cargo.md

```diff
@@ -0,0 +1,117 @@
+# Setting up your Cargo.toml file for ICU4X
+
+ICU4X makes heavy use of small crates and Cargo features in order to be highly modular. This tutorial is intended to help you set up a Cargo.toml file to download what you need for ICU4X.
+
+## Basic Cargo.toml with testdata
+
+The most basic Cargo.toml to get you off the ground is the following:
+
+```toml
+[package]
+name = "demo_testdata"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+icu = "1.1"
+icu_testdata = "1.1"
+```
+
+In your main.rs, you can use most functionality of ICU4X.
```

Comment:
'most' is pretty handwavy. You can use all stable functionality, for some locales (link to test locales?).

Reply:
You also can't use certain features that are not re-exported, like most of the data provider crates.